src/components/Notifications/Notifications.tsx

```diff
@@ -454,7 +478,9 @@ export class TurnIndicator extends React.Component<{}, any> { /* {{{ */
     constructor(props) {
         super(props);
         this.state = {
-            count: Object.keys(notification_manager.boards_to_move_on).length
+            count: Object.keys(notification_manager.boards_to_move_on).length,
+            total: Object.keys(notification_manager.active_boards).length,
```

Comment:
You assign a value to `total` only in the constructor. At this point `total` is always `0`.

For `count` it was done with: 
https://github.com/online-go/online-go.com/blob/aa1fc2028307381aae0aeb427629f0992f20fa17/src/components/Notifications/Notifications.tsx#L36
https://github.com/online-go/online-go.com/blob/aa1fc2028307381aae0aeb427629f0992f20fa17/src/components/Notifications/Notifications.tsx#L360
https://github.com/online-go/online-go.com/blob/aa1fc2028307381aae0aeb427629f0992f20fa17/src/components/Notifications/Notifications.tsx#L488-L490

Reply:
I don't think this is the case, it is still updating somehow (though don't ask me why) because the "it's your turn to move" counter still works properly. Sorry if I wasn't clear but that half still renders, the only thing the doesn't render is the 0 in the circle when you have active games but not any that are your turn...

Reply:
when you change 
`                 <span className={this.state.total > 0 ? "active count" : "count"}><span>{this.state.count}</span></span>`
to
`                 <span className={this.state.total > 0 ? "active count" : "active count"}><span>{this.state.total}</span></span>`

you can see that `total` is always `0`.

Reply:
ok yes I see that you're right about this point and I will fix it now, but I can't see how it would cause the specific rendering issues I'm having... 😕

Reply:
well I still have no idea why, but that magically fixed everything :D haha it's not working perfectly 😆 

Thanks a lot @flovo ❤️